modules/openapi-generator/src/main/resources/python/pyproject.mustache

```diff
@@ -32,10 +32,16 @@ typing-extensions = ">=4.7.1"
 pytest = ">=7.2.1"
 tox = ">=3.9.0"
 flake8 = ">=4.0.0"
+types-python-dateutil = ">=2.8.19.14"
+mypy = "1.4.1"
+
 
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [tool.pylint.'MESSAGES CONTROL']
 extension-pkg-whitelist = "pydantic"
+
+[tool.mypy]
+files = ["{{{packageName}}}"]
```

Comment:
Should we include tests right now as well?

Reply:
I really haven't checked the tests at this stage, I'll take a look!

Reply:
I took a look at the autogenerated tests in `test/` and this will require a bunch more work:

* The tests are not autogenerated/overwritten (as you mentioned elsewhere ;) )
* They are not really up to date, sadly. In particular, they use the non-aliased versions of all the fields. I only took a quick look but this seems it will require extensive change in the Java code generator to process all the fields towards their "aliased" version.

Also, it seems most of the autogenerated tests are actually not doing anything useful as this stage.

I will enable if for the "hand-written" tests from `tests/` though (it actually already found some problems).

Reply:
Done now :+1:

Reply:
Perfect, hand-written tests are probably most important anyway. Thanks!

Reply:
Actually, there's already another issue about the non-working default tests: https://github.com/OpenAPITools/openapi-generator/issues/5377